.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,11 +1,21 @@
 ## Summary
+<!-- A short, clear statement of the proposed changes. -->
 
-Summarize your PR. If it involves visual changes include a screenshot or gif.
+## How to test
+<!-- Include steps for testing your PR and screenshots (if applicable). -->
 
+## Risk matrix
+<!--
+Provide a matrix or statement of known risks and how they are mitigated.
+See risk examples: https://github.com/elastic/kibana/blob/main/RISK_MATRIX.mdx
+-->
+
+| Risk                      | Probability | Severity | Mitigation/Notes        |
+|---------------------------|-------------|----------|-------------------------|
```

Comment:
I think the risk matrix is useful, but having a full example here is overkill and just creates toil. Realistically you rarely need to see the examples more than once, so I moved them to the RISK_MATRIX file.